Userland/Applications/SoundPlayer/VisualizationWidget.h

```diff
@@ -62,7 +61,7 @@ class VisualizationWidget : public GUI::Frame {
         if (buffer_position + m_render_buffer.size() >= m_sample_buffer.size())
             buffer_position = m_sample_buffer.size() - m_render_buffer.size();
 
-        AK::TypedTransfer<float>::copy(m_render_buffer.data(), m_sample_buffer.span().slice(buffer_position).data(), m_render_buffer.size());
+        memcpy(m_render_buffer.data(), m_sample_buffer.span().slice(buffer_position).data(), m_render_buffer.size() * sizeof(float));
```

Comment:
This looks like it can use normal span copying, no need for memcpy

Reply:
This is not equivalent to a normal span copy because as far as I can see, there's no requirement in this code that the two spans are equal size. Span::copy_to verifies that fact.

Reply:
Then I think we need a better wrapper (copy_shortest?), or keep the TypedTransfer usage. sizeof(blah) is generally a code smell.

Reply:
Does this need another function? Wouldn't this be equivalent to `m_sample_buffer.span().slice(buffer_position, m_render_buffer.size())`?